circle.yml

```diff
@@ -17,6 +17,9 @@ test:
     - docker run -d --name airwatch --link mongo:mongo -e ROOT_URL=http://localhost -p 80:80 tutum.co/sjmatta/airwatch
     - sleep 25s
     - curl --retry 10 --retry-delay 5 -v http://localhost
+    - docker login -e "$DOCKERHUB_EMAIL" -u "$DOCKERHUB_USER" -p "$DOCKERHUB_API"
```

Comment:
Move lines 20-22 to deployment below. We only want to push to Docker when the release branch is built.